### ISSUE TITLE
green bar does not disappear in case of error message

### ISSUE THREAD

**Issue by volodymyrss** [oda-hub/frontend-astrooda#64]
frontend recieved this in the dispatcher rsponse:

```json
{"exit_status":
  {"comment":"",
   "debug_message":"",
   "error_message":"DDAException('AnalysisException:{}')",
   "job_status":"unknown",
   "message":"failed: isgri_image",
   "status":1,
   "warning":""}
}
```

the message is shown as expected, by the green bar stays

![image](https://user-images.githubusercontent.com/3909535/124890925-cec6d800-dfd8-11eb-9bb7-60e491e4ef7a.png)

**Comment by burnout87** [comment]
Better now?

![Screenshot from 2021-07-12 14-52-01](https://user-images.githubusercontent.com/3757833/125290632-d1029c80-e320-11eb-9327-df5babece23c.png)

**Comment by volodymyrss** [comment]
I still observe this:

![image](https://user-images.githubusercontent.com/3909535/125292204-84b85c00-e322-11eb-8fe9-0bf4a987ae2b.png)

Note that the error is a bit different, but it is possible it would have been fixed for both cases.

Which commit addresses it?

https://frontend.staging.odahub.io/cdci/astrooda/ uses 
https://github.com/oda-hub/frontend-astrooda/commit/c4eef38f140b7bd8ce6558804c1b2a09085be2bb

which is the last available

**Comment by burnout87** [comment]
Sorry, I didn't specify. It's locally on my machine, will merge the request now.

**Comment by volodymyrss** [comment]
> Sorry, I didn't specify. It's locally on my machine, will merge the request now.
> 
> It was just to double-check if that is the expected behavior

It looks as expected, but indeed, it is not exactly the situation experienced (different error). I check when pushed!

**Comment by burnout87** [comment]
Merged now on gitlab.

> [c4eef38](https://github.com/oda-hub/frontend-astrooda/commit/c4eef38f140b7bd8ce6558804c1b2a09085be2bb)

This should fix #65 , but anyway to be verified on the running instance

**Comment by volodymyrss** [comment]
yeah, this is ok
![image](https://user-images.githubusercontent.com/3909535/125296825-f72b3b00-e326-11eb-862d-fc3ad5fe8c56.png)

thanks!
